services/app-api/forms/mcpar.json

```diff
@@ -1476,34 +1539,42 @@
                     "validation": "checkbox",
                     "props": {
                       "label": "C2.V.7 Monitoring methods",
-                      "hint": "What are the monitoring methods? Check all that apply or add free text if you select “Other”.",
+                      "hint": "Select the method(s) utilized to assess compliance with the state's 42 C.F.R. § 438.68 and 42 C.F.R. § 438.206 standards. If a method is not listed, select \"Custom Method\" and add details.",
```

Comment:
nit: do we want smart quotes around "Custom Method" here like we usually do for quotes in text?